lib/envars.mli

```diff
@@ -76,7 +76,4 @@ val xdg_data_dirs   : (string -> unit) -> string list
 val xdg_dirs : warn : (string -> unit) -> string list
 
 (** {6 Prints the configuration information } *)
-val print_config : ?prefix_var_name:string -> out_channel -> unit
-
-(** Directories in which coq sources are found *)
-val coq_src_subdirs : string list
+val print_config : ?prefix_var_name:string -> out_channel -> string list -> unit
```

Comment:
This is not OK in my humble opinion. In particular the print_config should not take a string list (with no label, no comments...). The config is Coq_config, potrentially overridden by env variables.  This third side-channel to change the config is a bit messy.

I see 2 options:
- pass to print_config the bypass_API boolean option, and have the function print the right set of dirs taken from Coq_config
- have print confing print all class of src_ directories, have coq_makefile put in the conf file if the API is bypassed, and finally have CoqMakefile.in do the if and use one or the other set of directories